src/main/java/frc/robot/subsystems/tank/TankSubsystem.java

```diff
@@ -20,6 +30,12 @@
   private final WPI_TalonSRX rightMain;
   private final WPI_TalonSRX rightFollow;
 
+  private final AHRS ahrs;
+
+  private final DifferentialDrivePoseEstimator odometry;
+
+  public static final double ENCODER_TICKS_TO_INCHES = 32 / 142.40;
```

Comment:
Fortunately you have the `.40` in this equation, otherwise the result is `0`. Ask me how I know and how many hours I spent hunting for this bug.

If you have integer division, you get an integer result.